kindlecomicconverter/KCC_gui.py

```diff
@@ -918,6 +913,16 @@ def __init__(self, kccapp, kccwindow):
             if self.windowSize == '0x0':
                 MW.resize(500, 500)
 
+        self.formats = {  # text, icon, data/option_format
+            "MOBI/AZW3": {'icon': 'MOBI', 'format': 'MOBI'},
+            "EPUB": {'icon': 'EPUB', 'format': 'EPUB'},
+            "CBZ": {'icon': 'CBZ', 'format': 'CBZ'},
+            "EPUB (Calibre KFX)": {'icon': 'EPUB', 'format': 'KFX'},
```

Comment:
Should this be `KFX.png`?

Reply:
As you said, the result is not really a kfx file but an epub that needs to be converted to kfx in a second time. 
That's why I left an epub icon.

Reply:
Alright just confirming. LGTM